library/src/main/java/org/apache/apex/malhar/lib/function/Function.java

```diff
@@ -35,7 +35,7 @@
  * @since 3.4.0
  */
 @InterfaceStability.Evolving
-public interface Function
+public interface Function extends java.io.Serializable
```

Comment:
Kryo cannot load a lambda class. Therefore the problem described below applies, even if the annotation is recognized, which I'm not sure about either.